src/cmd/describegpt.rs

```diff
@@ -42,15 +45,28 @@ struct Args {
     flag_tags:        Option<bool>,
     flag_max_tokens:  Option<i32>,
     flag_json:        Option<bool>,
+    flag_user_agent:  Option<String>,
 }
 
 // OpenAI API model
 const MODEL: &str = "gpt-3.5-turbo-16k";
 
-fn get_completion(api_key: &str, messages: &serde_json::Value, max_tokens: Option<i32>) -> String {
+fn get_completion(
+    api_key: &str,
+    messages: &serde_json::Value,
+    max_tokens: Option<i32>,
+    flag_user_agent: Option<String>,
+) -> String {
+    // If --user-agent is not specified, use an empty string as the default value
```

Comment:
Hi @rzmk , you don't need to do this. Just set user_agent when setting up the reqwest client...